src/ngAutocomplete.js

```diff
@@ -148,6 +148,12 @@ angular.module( "ngAutocomplete", [])
           }
         }
 
+        if (scope.options.getPlaceOnBlur) {
+          element.bind('blur', function () {
```

Comment:
When user click on dropdown proposals to choose a place:
- `blur` event is fired
- calling unexpected `getPlace()`

I can't find a way to check if input is blured while clicking results list dropdown to prevent it.